src/Bicep.LangServer/Snippets/Templates/res-log-analytics-solution.bicep

```diff
@@ -1,17 +1,21 @@
 ﻿// Log Analytics Solution
-resource /*${1:logAnalyticsSolution}*/logAnalyticsSolution 'Microsoft.OperationsManagement/solutions@2015-11-01-preview' = {
+resource /*${1:operationalInsightsWorkspace}*/operationalInsightsWorkspace 'Microsoft.OperationalInsights/workspaces@2021-06-01' existing = {
   name: /*${2:'name'}*/'name'
+}
+
+resource /*${3:logAnalyticsSolution}*/logAnalyticsSolution 'Microsoft.OperationsManagement/solutions@2015-11-01-preview' = {
+  name: /*${4:'name'}*/'name'
   location: resourceGroup().location
   properties: {
-    workspaceResourceId: resourceId('Microsoft.OperationalInsights/workspaces', /*${3:'logAnalyticsWorkspace'}*/'logAnalyticsWorkspace')
+    workspaceResourceId: operationalInsightsWorkspace.id
     containedResources: [
-      resourceId('Microsoft.OperationalInsights/workspaces/views', /*${3:'logAnalyticsWorkspace'}*/'logAnalyticsWorkspace', /*${4:'logAnalyticsSolution'}*/'logAnalyticsSolution')
+      /*${5:'id'}*/'id'

```

Comment:
I think you can go with one of either:
```bicep
/*${5:viewRef}*/viewRef.id
```
Or:
```bicep
/*${5:viewId}*/viewId
```

Reply:
Updated here:
https://github.com/Azure/bicep/commit/c4c04eb3bc9a0d489686c47d104a3ee22178b142